staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -95,7 +95,6 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 	}
 	// The list of filesystems that require containerized mounter on GCI image cluster
 	fsTypesNeedMounter := map[string]struct{}{
-		"nfs":       {},
```

Comment:
Since this is in the general mount library, I wonder how many users have started to depend on this and may break.

Reply:
containerized mounter was implemented for gci at that time due to the lack of mount util. are there any other users need this too?

Reply:
It's hard to say because it's not in a gce specific folder, it's in the general mount library, so anybody can use it.

The entire containerized mounter is targeted to be removed in 1.23.